bundles/org.openhab.automation.luascripting/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.automation.luascripting</artifactId>
+
+  <name>openHAB Add-ons :: Automation :: Lua Scripting</name>
+
+  <properties>
+    <bnd.fixupmessages><![CDATA["Classes found in the wrong directory","The default package '.' is not permitted by the Import-Package syntax"; restrict:=error; is:=warning]]></bnd.fixupmessages>
+
+    <bnd.importpackage>
+      com.ibm.icu.*;resolution:=optional,
+      org.abego.treelayout.*;resolution:=optional,
+      org.apache.ivy.*;resolution:=optional,
+      org.stringtemplate.v4.*;resolution:=optional</bnd.importpackage>
```

Comment:
Using multiple lines is more readable, but unfortunately it results in some build warnings, see: #9460

Reply:
I only added bnd.importpackage since it seemed to be the only way for the binding to build. But after a bit of trial and error I was able to get rid of those imports entirely.